### PR TITLE
Addition of histograms for SiStrip : Cluster widths vs amplitudes

### PR DESCRIPTION
**Pull request by hdelanno**

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
ciao @hdelanno 
apparently we are unlucky with the tests. we will have to wait..
in the meantime: are 200k bins really needed for the 2D monitor element you add? if some optimization can be done, please go ahead.
thanks,
F.

**Comment by deguio** [comment]
please test

**Comment by hdelanno** [comment]
Hello @deguio 
Ah, it finally worked !
About the 200k bins, I have to discuss this with Tomas Hreus who asked me to add those plots. He's on vacation until the beginning of August. I'll ask him as soon as he gets back. But I guess we could divide the number of bins by four or something like that.
Thanks,
Hugo

**Comment by deguio** [comment]
that would be nice. thanks @hdelanno 
F.

**Comment by deguio** [comment]
@hdelanno 
please let me know if an additional commit is coming for this PR.
thanks,
F.

**Comment by hdelanno** [comment]
Hi @deguio ,
I'm sorry for the time it takes. Still waiting for the answer of Tomas. I'll update the commit as soon as I have feedback for the "best optimisation" for the number of bins!
Best,

Hugo

**Comment by deguio** [comment]
-1
rejecting for now. feel free to update the same branch and we will consider this PR again.
thanks for understanding,
F.

**Comment by hdelanno** [comment]
Hi Federico,
I'm really sorry for the delay.
I will update the branch as soon as I have news.
Best,
Hugo

File: DQM/SiStripMonitorCluster/python/SiStripMonitorCluster_cfi.py [hunk 1]
```diff
@@ -271,6 +271,19 @@
...
+        Nbinsx = cms.int32(2000),
+        xmin = cms.double(0.),
+        xmax = cms.double(2000.),
+        Nbinsy = cms.int32(100),
+        ymin = cms.double(0.),
+        ymax = cms.double(100.),
+        globalswitchon = cms.bool(False),
+        subdetswitchon = cms.bool(True), 
+        layerswitchon  = cms.bool(True),
+        moduleswitchon = cms.bool(False)
```
Comment (threus) [inline DQM/SiStripMonitorCluster/python/SiStripMonitorCluster_cfi.py]: @hdelanno if we set only the globalswitchon to True and others to False, we keep a single plot in the DQM, still enough for routine checks and if need be, one can reprocess privately with all plots alive. Could you please set subdet/layer to False and global to True? That should save a lot of space.
Sorry for my delayed comment.

**Comment by hdelanno** [comment]
Hi @deguio 
As you can see in the comment of @threus we'd like to keep the number of bins as it is, but we have changed the subdet/layer switches to False in order to only have one plot displayed (and we can reprocess locally with all plots on if needed for investigating).
If it is ok for you, could you reconsider this request? Thank you very much!

**Comment by danduggan** [comment]
please test

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**